openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -27,13 +28,28 @@
 PRIMITIVE_NEO4J_TYPES = (integer, string, neo4j_unicode, float, bool)
 
 
+@task
+def method_wrapper(method, *args):
+    """
```

Comment:
Just a nit, for consistency please use http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#id9 for doc strings.